### PR TITLE
Add reusable swap CI

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: .github/workflows/swap-ci-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - master
+    - main
+    - develop
+  pull_request:
+
+jobs:
+  job_functional_tests:
+    uses: LedgerHQ/app-exchange/.github/workflows/reusable_swap_functional_tests.yml@develop
+    with:
+      branch_for_bitcoin: ${{ github.ref }}
```
Comment (xchapron-ledger) [inline .github/workflows/swap-ci-workflow.yml:16]: to change?

**Review by xchapron-ledger — APPROVED** [review]

File: .github/workflows/swap-ci-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    - master
+    - main
+    - develop
+  pull_request:
+
+jobs:
+  job_functional_tests:
+    uses: LedgerHQ/app-exchange/.github/workflows/reusable_swap_functional_tests.yml@develop
+    with:
+      branch_for_polkadot: ${{ github.ref }}
```
Comment (xchapron-ledger) [inline .github/workflows/swap-ci-workflow.yml:16]: Maybe `repo_for_polkadot` could be added too?

Comment (xchapron-ledger) [reply .github/workflows/swap-ci-workflow.yml:16]: https://github.com/LedgerHQ/app-tezos-new/blob/develop/.github/workflows/swap.yml#L17

Comment (fbeutin-ledger) [reply .github/workflows/swap-ci-workflow.yml:16]: Good catch, thanks

**Pull request merged**